packages/data-point/examples/entity-request-options-override.js

```diff
@@ -4,9 +4,6 @@ const mockRequest = require('./entity-request-basic.mock')
 dataPoint.addEntities({
   'request:getLuke': {
     url: 'https://swapi.co/api/people/1/'
-  },
-  params: {
-    inspect: false
```

Comment:
Oh I should have caught that -- the params object should have been inside the `request` definition:

```js
dataPoint.addEntities({
  'request:getLuke': {
    url: 'https://swapi.co/api/people/1/',
    params: {
      inspect: true
    }
  }
})
```